src/cheri/app-cap-description.adoc

```diff
@@ -243,44 +262,12 @@ implementation supports {cheri_default_ext_name} _and_ <<x_perm>> is set.
 . For MXLEN=64, the bit assigned to the <<m_bit>> must be zero if <<x_perm>> isn't set.
 . For MXLEN=32, the <<m_bit>> is only encoded in quadrant 1 and does _not_ exist in the other quadrants.
 
-[#sec_permission_transitions]
-===== Permission Transitions
-
-Executing <<CLRPERM>> can result in sets of permissions which cannot be represented when MXLEN=32
-(see <<cap_perms_encoding32>>) or permission combinations which are not useful for MXLEN=64, such as
-<<asr_perm>> set without <<x_perm>>.
-These cases are defined to return useful minimal sets of permissions.
-
-When removing permissions with <<CLRPERM>> the following rules,
-along with any added by extensions,
-are iterated to fixed-point:
-
-[#acperm_rules]
-.<<CLRPERM>> common rules
-[float="center",align="center",cols="2,2,4",options="header"]
-|===
-| Rule            | Permission   | Valid only if
-|  1  (RV32 only) | <<asr_perm>> | All other permissions are set.
-|  2              | <<c_perm>>   | <<r_perm>> or <<w_perm>>
-|  3  (RV32 only) | <<c_perm>>   | <<r_perm>>
-|  4  (RV32 only) | <<x_perm>>   | <<r_perm>>
-|  5  (RV32 only) | <<w_perm>>   | not(<<c_perm>>) or <<lm_perm>>
-|  6  (RV32 only) | <<x_perm>>   | <<w_perm>> or <<c_perm>>
-|  7              | <<lm_perm>>  | <<c_perm>> and <<r_perm>>
-|  8  (RV32 only) | <<lm_perm>>  | (<<w_perm>> or <<zylevels1_lg_perm>>)
-|  9              | <<asr_perm>> | <<x_perm>>
-| 10^1^           | <<m_bit>>    | <<x_perm>> and {cheri_default_ext_name} is implemented
-|===
-
-^1^ For RV32, the encodings which have the <<m_bit>> set to {int_mode_value} for {cheri_int_mode_name}
-are only valid if {cheri_default_ext_name} is implemented. Otherwise those encodings represent invalid permissions.
-
 NOTE: Future extensions may allow more combinations of permissions, especially for MXLEN=64.
 
 NOTE: Future extensions may define new dependent permissions and, if so, must augment the above table.
 
-If any of these transitions (or those added by extensions or the encoding) are applied to a sealed capability,
-the {ctag} of the resulting capability is cleared.
+If any of these transitions (or those added by extensions or the encoding) alter a sealed capability,
```

Comment:
we got complaints about referring to capabilities as variables - so I think we need to say "if the source capability is sealed"